$OpenBSD: patch-glib_gutils_c,v 1.4 2018/09/24 20:00:59 jasper Exp $

/usr/share is not a valid data directory on OpenBSD for xdg/glib purposes

Index: glib/gutils.c
--- glib/gutils.c.orig
+++ glib/gutils.c
@@ -2040,7 +2040,7 @@ g_get_system_data_dirs (void)
 
 #ifndef G_OS_WIN32
       if (!data_dirs || !data_dirs[0])
-          data_dirs = "/usr/local/share/:/usr/share/";
+          data_dirs = "/usr/local/share/";
 
       data_dir_vector = g_strsplit (data_dirs, G_SEARCHPATH_SEPARATOR_S, 0);
 #else
